guides/operator-guide.md

```diff
@@ -10,9 +10,31 @@ RIG uses environment variables for most of its configuration -- they're listed i
 
 Variable | Description | Default
 -------- | ----------- | -------
+`API_PORT` | Port at which RIG exposes internal APIs such as API Proxy management or user connections management. | 4010
+`DISCOVERY_TYPE` | Type of discovery used in distributed mode. If not set discovery is not used. Available options: `dns`. | nil
+`DNS_NAME` | Address where RIG will do DNS discovery for Node host addresses. | "localhost"
+`HOST` | Hostname for Phoenix endpoints (HTTP communication). | "localhost"
+`INBOUND_PORT` | Port at which RIG exposes proxy and websocket/sse communication. | 4000
+`JWT_BLACKLIST_DEFAULT_EXPIRY_HOURS` | Default expiration time in hours for blacklisted JWTs. Used if JWT doesn't have an expiration time in claims. | 1
+`JWT_ROLES_FIELD` | Key in JWT claims under which roles are set for each user. | "roles"
 `JWT_SECRET_KEY` | The secret key used to sign and verify the JSON web tokens. | ""
 `JWT_USER_FIELD` | The JSON web token as sent by the front-ends should contain the user ID, in the same format used by the back-ends in the messages they send towards the user. `JWT_USER_FIELD` is the name of that user ID field in the JWT. For the corresponding field used in outbound messages, see `MESSAGE_USER_FIELD`. | "user"
-`MESSAGE_USER_FIELD` | (Outbound) messages are expected to be in JSON format. For routing the message to a specific user, RIG expects the user's ID to be present in such a JSON message. The corresponding JSON field is defined by `MESSAGE_USER_FIELD`. | "user"
+`KAFKA_CONSUMER_GROUP` | Consumer group name for Kafka. | "rig-consumer-group"
 `KAFKA_ENABLED` | If enabled, RIG will consume messages from a Kafka broker using the configured broker and topic(s). | false
+`KAFKA_HOSTS` | List of Kafka brokers RIG should connect to, delimited by comma. Usually it's enough to specify one broker and RIG will auto-discover rest of the Kafka cluster. | ["localhost:9092"]
+`KAFKA_LOG_TOPIC` | Kafka topic for producer used to log HTTP requests going through RIG's API Proxy. | "rig-request-log"
+`KAFKA_SOURCE_TOPICS` | List of Kafka topics RIG will consume, delimited by comma. | ["rig"]
+`MESSAGE_USER_FIELD` | (Outbound) messages are expected to be in JSON format. For routing the message to a specific user, RIG expects the user's ID to be present in such a JSON message. The corresponding JSON field is defined by `MESSAGE_USER_FIELD`. | "user"
+`NODE_COOKIE` | Erlang cookie used in distributed mode, so nodes in cluster can communicate between each other. | nil
+`NODE_HOST` | Erlang hostname for given node thanks to which is routable by other nodes in cluster. | nil
```

Comment:
can you make "thanks to which is routable" a bit clearer?

Perhaps it would be good to mention that this is used to build the Erlang long-name `rig@NODE_HOST`